tests/state/test_particle.py

```diff
@@ -48,3 +48,17 @@ def test_create_antiparticle(particle_name, anti_particle_name):
     assert anti_particle.width == comparison_particle.width
     assert anti_particle.state == comparison_particle.state
     assert anti_particle.name == "anti-" + particle_name
+
+
+def test_create_antiparticle_tilde():
+    anti_particles = DATABASE.find_subset("~")
+    assert len(anti_particles) == 9
+    for anti_particle in anti_particles.values():
+        particle_name = anti_particle.name.replace("~", "")
+        particle = DATABASE[particle_name]
+        anti_particle_name = particle_name + "~"
+        anti_particle_name = anti_particle_name.replace("0~", "~0")
+        created_anti_particle = create_antiparticle(
+            particle, anti_particle_name
```

Comment:
You could also just create the particle from antiparticle directly and compare

Reply:
No, because then the names would have been unequal. The `create_antiparticle` function prepends `"anti-"` if you don't provide a name.

Reply:
cant you provide particle_name as a name?